### ISSUE TITLE
Regarding val_acc and val_loss

### ISSUE THREAD

**Issue by RakeshSahoo90** [SadmanSakib93/Stratified-k-fold-cross-validation-Image-classification-keras#7]
Sir, How to add val_accuracy and val_loss of each epoch with the existing training loss, and training accuracy?
Can u please update us about the K-fold cross validation

**Comment by SadmanSakib93** [comment]
To add the validation loss/accuracy, the following approach can be used within the stratified cross-validation code:

```
validation_generator = validation_datagen.flow_from_directory(
        validation_path,
        target_size=(img_rows, img_cols),
        batch_size=batch_size,
        class_mode='categorical',  # only data, no labels
        )

# fit model
history = model.fit(train_generator,
                        epochs=epoch,
                        validation_data=validation_generator)
```

**Comment by RakeshSahoo90** [comment]
Thank you Sir
